pandas/tests/formats/test_format.py

```diff
@@ -3571,6 +3571,67 @@ def test_to_latex_multiindex(self):
 
         self.assertEqual(result, expected)
 
+    def test_to_latex_multicolumnrow(self):
+        df = pd.DataFrame({
+                    ('c1',0):dict((x,x) for x in range(5)),
+                    ('c1',1):dict((x,x+5) for x in range(5)),
+                    ('c2',0):dict((x,x) for x in range(5)),
+                    ('c2',1):dict((x,x+5) for x in range(5)),
+                    ('c3',0):dict((x,x) for x in range(5))
+                    })
+        result = df.to_latex(multicolumn=True)
```

Comment:
Further, can you add test cases with and without index/column level names? (I think now all are without?) Or is this already tested somewhere else?